frontend/src/v4/routes/teamspaces/components/uploadModelFileDialog/uploadModelFileDialog.component.tsx

```diff
@@ -179,11 +227,13 @@ export class UploadModelFileDialog extends React.PureComponent<IProps, IState> {
 
 		if (extension === 'spm') {
 			this.setState({
-				allowImportAnimations: true
+				allowImportAnimations: true,
+				allowSetTimezone: true
 			}, () => setFieldValue('importAnimations', true));
 		} else if (this.state.allowImportAnimations) {
 			this.setState({
-				allowImportAnimations: false
+				allowImportAnimations: false,
+				allowSetTimezone: false
```

Comment:
I think this should be a separate else if?

or we just do else without the condition